### PR TITLE
run travis for both rack versions

### PR DESCRIPTION
**Pull request by mathieujobin**
also remove activesupport and sqlite3 from dev dependencies

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,4 +1,16 @@
 rvm:
-  - 2.0.0-p353
+  - 2.0.0
```
Comment (mathieujobin) [inline .travis.yml]: greenfield is on the latest -p648

File: api_hammer.gemspec [hunk 2]
```diff
@@ -34,6 +34,4 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'minitest-reporters'
   spec.add_development_dependency 'yard'
   spec.add_development_dependency 'simplecov'
-  spec.add_development_dependency 'activesupport'
-  spec.add_development_dependency 'sqlite3'
```
Comment (notEthan) [inline api_hammer.gemspec]: hm, could have sworn I used sqlite somewhere in the code, but apparently not

File: api_hammer.gemspec [hunk 3]
```diff
@@ -34,6 +34,4 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'minitest-reporters'
   spec.add_development_dependency 'yard'
   spec.add_development_dependency 'simplecov'
-  spec.add_development_dependency 'activesupport'
```
Comment (notEthan) [inline api_hammer.gemspec]: activesupport, however, is needed - why is this removed?

Comment (notEthan) [inline api_hammer.gemspec]: oh wait, it isn't here ... that was a different branch. I do need activesupport though.

**Comment by mathieujobin** [comment]
@notEthan put back active support

**Comment by mathieujobin** [comment]
updated the gemfiles to precise the versions of activesupport, as we can't install 5 on ruby 2.0 for example. I moved the two gemfiles under a gemfiles/ folder, I found that to be more standard accros gems I've been looking at lately

there is one failure left though... after including activesupport 5, some method becomes projected.
I guess it wasn't before... I don't know enough to look at this tonight, i think I will leave that one to you.

**Comment by mathieujobin** [comment]
@notEthan this is ready now ! :money_with_wings: 
strangely that single failure I mentioned only happens for ruby 2.3.1, and it pass for ruby-head (which will be released as 2.4.0 on Dec 25th

**Comment by notEthan** [comment]
thanks @mathieujobin
